CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Unreleased
 
+<<<<<<< HEAD
```

Comment:
This looks like a merge conflict

Reply:
Oi! Thanks, strangely `git status` didn't told me.